ft_headmovement.m

```diff
@@ -11,12 +11,34 @@
 %   cfg.trl          = Nx3 matrix with the trial definition, see FT_DEFINETRIAL
 %   cfg.numclusters  = number of segments with constant headposition in which to split the data (default = 12)
 %
-% This method and related methods are described by Stolk et al., Online and
+% optional arguments are
+%   cfg.updatesens   = 'yes' (default) or 'no'
+%   cfg.avgovertrial = 'no' (default) or 'yes'
+%   cfg.pertrial     = 'no' (default) or 'yes'
+%
+% If cfg.updatesens is specified, the output grad-structure has a 
+% specification of the coils expanded as per the centroids of the position
+% clusters. The balancing matrix is s a weighted concatenation of the
+% original tra-matrix. If cfg.updatesens is false, the output grad is a
+% struct-array with coil and sensor positions as per the clusters'
+% centroids. If cfg.avgovertrial is specified, the clustering of head
+% positions is done on the average headcoil-position per trial.
+%
+% Additional outputs are:
```

Comment:
Multiple low-level outputs is not really FT-style. Looking at ft_regressconfound I see cfg.output=residual/model/beta. Idem for ft_mvaranalysis.

How about adding
cfg.output=original/averaged/split
and adding the trialinfo elsewhere (?)

Reply:
Fair point. I will give it some thought. Perhaps, the 'original' can go altogether, and the average/split option can be used. Also, I realize that for extreme consistency (and for ft_analysispipeline to be able to work) we should probably add a cfg field to the grad-structures, in which implicitly (e.g. in cfg.previous.trl in combination with cfg.trials) the 'trialinfo' can be represented.

Reply:
regarding grad.cfg, after ft_electroderealign the output elec does get a cfg.